src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -417,8 +422,8 @@ worker_apply_shard_ddl_command(PG_FUNCTION_ARGS)
 
 	/* extend names in ddl command and apply extended command */
 	RelayEventExtendNames(ddlCommandNode, schemaName, shardId);
-	ProcessUtility(ddlCommandNode, ddlCommand, PROCESS_UTILITY_TOPLEVEL,
-				   NULL, None_Receiver, NULL);
+	CitusProcessUtility(ddlCommandNode, ddlCommand, PROCESS_UTILITY_TOPLEVEL, NULL,
+						None_Receiver, NULL);
```

Comment:
Not this commit's fault, but why on earth is it correct to use PROCESS_UTILITY_TOPLEVEL here?  Isn't that what makes it easy to crash citus by sending a CONCURRENTLY command w/ worker_apply_shard_ddl_command?